rust/osm2lanes/src/road/lane.rs

```diff
@@ -158,3 +161,20 @@ impl Printable for Direction {
         }
     }
 }
+
+/// Access by vehicle type
+/// Types as defined in <https://wiki.openstreetmap.org/wiki/Key:access#Land-based_transportation>
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "type", rename_all = "snake_case")]
+pub struct Access {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    foot: Option<AccessValue>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    bicycle: Option<AccessValue>,
```

Comment:
The only case that may come up from A/B Street in the medium-term is bicycle=dismount. But just something to keep in mind, this all seems fine to start